ray-operator/controllers/ray/utils/constant.go

```diff
@@ -167,6 +168,9 @@ const (
 
 	// RayNodeHeadGroupLabelValue is the value for the RayNodeGroupLabelKey label on a head node
 	RayNodeHeadGroupLabelValue = "headgroup"
+
+	// Telemetry
+	KUBERAY_VERSION = "nightly"
```

Comment:
@kevin85421 since this is defined as a constant, it will always be `nightly` instead of the actual kuberay version. Is that expected or am I missing something?

Reply:
I opened a PR to the release branch to update it #2036.